containers/members/MemberStorageSelector.js

```diff
@@ -8,8 +8,10 @@ import humanSize from 'proton-shared/lib/helpers/humanSize';
 const FIVE_GIGA = 5 * GIGA;
 
 const MemberStorageSelector = ({ member, organization, onChange }) => {
-    const minPadding = 0;
-    const maxPadding = organization.MaxSpace - organization.AssignedSpace;
+    const step = GIGA;
+    const minPadding = Math.ceil(member.UsedSpace / GIGA) * GIGA;
+    const maxPadding =
+        Math.ceil((organization.MaxSpace - organization.UsedSpace + member.UsedSpace) / GIGA) * GIGA + step;
```

Comment:
should it be `organization.MaxSpace - organization.UsedSpace + member.MaxSpace`?

Reply:
Yeah, I know it's a bit confusing, but it's not the same logic between both selectors. What I wrote is correct.